website/news.html

```diff
@@ -19,6 +19,10 @@
 <script>
 document.addEventListener('DOMContentLoaded', () => {
   const items = [
+    {
+      date: new Date('2023-05-17'),
+      text: 'Vision paper on modern cloud development accepted to appear at <a href="https://sigops.org/s/conferences/hotos/2023/">HotOS\'23</a>. You can read the draft <a href="./assets/docs/hotos23_vision_paper.pdf">here</a>.',
```

Comment:
Replaced `accepted to appear at` with just `to appear at`?

Reply:
Done